Signal/src/ViewControllers/ConversationView/ConversationInputToolbar.m

```diff
@@ -241,7 +241,7 @@ - (void)ensureContentConstraints
             [self.attachmentView autoPinEdgeToSuperviewEdge:ALEdgeTop withInset:textViewVInset],
             [self.attachmentView autoPinEdgeToSuperviewEdge:ALEdgeBottom withInset:textViewVInset],
             [self.attachmentView autoPinEdgeToSuperviewEdge:ALEdgeLeft withInset:contentHInset],
-            [self.attachmentView autoSetDimension:ALDimensionHeight toSize:300.f],
+            [self.attachmentView autoSetDimension:ALDimensionHeight toSize:150.f],
```

Comment:
Height of attachment preview is pretty subjective.  WDYT?

Reply:
I built locally - this feels reasonable.